crypto.go

```diff
@@ -35,11 +35,9 @@ import (
 	"golang.org/x/crypto/pbkdf2"
 )
 
-const ed25519PrivateKeyPrefix = "302e020100300506032b657004220420"
-const ed25519PubKeyPrefix = "302a300506032b6570032100"
+const _Ed25519PrivateKeyPrefix = "302e020100300506032b657004220420"
 
-const _ECDSAPrivateKeyPrefix = "3030020100300706052b8104000a04220420"
-const _ECDSAPubKeyPrefix = "302f300706052b8104000a0324000421"
+const _ECDSAPrivateKeyPrefix = "30540201010420"
```

Comment:
If the user exports his ECDSA private key using OpenSSL and doesn't specify he wants the compressed format, the prefix will be `30740201010420`. Also the `parameters` and `publicKey` fields of `ECPrivateKey` are optional so there are a lot of valid ECDSA private key prefixes.

I think we can remove the `if strings.Contains(s, _ECDSAPrivateKeyPrefix)` and just call `_ECDSAPrivateKeyFromString`. And if there is an error, try `_Ed25519PrivateKeyFromString`